src/NLog.Web.AspNetCore/LayoutRenderers/AspNetResponseHttpsCompressionLayoutRenderer.cs

```diff
@@ -0,0 +1,61 @@
+﻿#if ASP_NET_CORE3
+using Microsoft.AspNetCore.Http.Features;
+using NLog.LayoutRenderers;
+using NLog.Web.Internal;
+using System;
+using System.Text;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// ASP.NET Response HTTPS Compression Mode
+    ///
+    /// Renders the Response HTTPS Compression Mode
+    ///
+    /// Compress - Opts into compression over HTTPS. Enabling compression on HTTPS requests for
+    /// remotely manipulable content may expose security problems.
+    ///
+    /// DoNotCompress - Opts out of compression over HTTPS. Enabling compression on HTTPS requests for
+    /// remotely manipulable content may expose security problems.
+    ///
+    /// Default - No value has been specified, use the configured defaults.
+    /// </summary>
+    /// <remarks>
+    /// ${aspnet-response-https-compression}
+    /// </remarks>
+    [LayoutRenderer("aspnet-response-https-compression")]
+    public class AspNetResponseHttpsCompressionLayoutRenderer : AspNetLayoutRendererBase
+    {
+        /// <summary>
```

Comment:
Can this be replaced by  `<inheritdoc/>` ?

Reply:
Complete